src/test/java/org/folio/rest/HeadersForwardingTest.java

```diff
@@ -119,6 +126,10 @@ public void testPostBlUsersLogin() {
         .withHeader(HttpHeaders.USER_AGENT, WireMock.matching(".+"))
         .withHeader(BLUsersAPI.X_FORWARDED_FOR_HEADER, WireMock.equalTo(IP))
     );
+
+    WireMock.getAllServeEvents().stream()
```

Comment:
there is some existing headers check logic line above, it would be good to perform extincting and yours headers check logic in one method

Reply:
header check logic in one method